src/stactools/landsat/constants.py

```diff
@@ -9,6 +9,13 @@ class Sensor(Enum):
     OLI_TIRS = "C"
 
 
+class GeometrySource(Enum):
+    USGS = "usgs"
+    FOOTPRINT = "footprint"
+    ANG = "ang"
+    BBOX = "bbox"
```

Comment:
I see that these enum choices are documented in the `create_item` docstring. I think either
1. That documentation should be moved to these enum choices, or
2. This enum's docstring should point the user to `create_item` for more information about what the choices mean

Reply:
Implemented #2.